tests/positive/gpu_av.cpp

```diff
@@ -19,6 +19,107 @@
 
 class PositiveGpuAssistedLayer : public VkGpuAssistedLayerTest {};
 
+TEST_F(PositiveGpuAssistedLayer, SetSSBOBindDescriptor) {
+    TEST_DESCRIPTION("Makes sure we can use vkCmdBindDescriptorSets()");
+    SetTargetApiVersion(VK_API_VERSION_1_1);
+    VkValidationFeaturesEXT validation_features = GetValidationFeatures();
+    ASSERT_NO_FATAL_FAILURE(InitFramework(m_errorMonitor, &validation_features));
+    if (!AreRequiredExtensionsEnabled()) {
+        GTEST_SKIP() << RequiredExtensionsNotSupported() << " not supported";
+    }
+    if (!CanEnableGpuAV()) {
+        GTEST_SKIP() << "Requirements for GPU-AV are not met";
+    }
+    ASSERT_NO_FATAL_FAILURE(InitState());
+    ASSERT_NO_FATAL_FAILURE(InitRenderTarget());
+
+    VkPhysicalDeviceProperties properties = {};
+    vk::GetPhysicalDeviceProperties(gpu(), &properties);
+    if (properties.limits.maxBoundDescriptorSets < 8) {
```

Comment:
nit: can just go 

```suggestion
    if (m_device->props.limits.maxBoundDescriptorSets < 8) {
```

and save the extra query